app/views/ConditionalYesNoPageView.scala.html

```diff
@@ -93,6 +95,16 @@ <h1 class="govuk-heading-l">@renderMessage(viewmodel.heading)</h1>
     )
 }
 
+@security(key: String, message: DisplayMessage) = @{
```

Comment:
I don't think we need this, we can just use the textArea to render this and add the hint

Reply:
We can use textArea but I am not sure because we have added field type of security for it